src/box/index.h

```diff
@@ -1071,7 +1092,9 @@ exhausted_index_read_view_iterator_next_raw(struct index_read_view_iterator *it,
 int
 generic_iterator_position(struct iterator *it, const char **pos,
 			  uint32_t *size);
-
+int
+generic_index_read_view_iterator_position(struct index_read_view_iterator *it,
+					  const char **pos, uint32_t *size);
```

Comment:
Nit: Please don't remove the empty line before `#if defined(__cplusplus)`.

Reply:
Done